lineman/app/components/deactivate_user_form/deactivate_user_form.coffee

```diff
@@ -0,0 +1,13 @@
+angular.module('loomioApp').factory 'DeactivateUserForm', ->
+  templateUrl: 'generated/components/deactivate_user_form/deactivate_user_form.html'
+  controller: ($scope, $rootScope, $window, CurrentUser, Records) ->
+    $scope.user = CurrentUser.clone()
+
+    $scope.submit = ->
+      $scope.isDisabled = true
+      Records.users.deactivate($scope.user).then ->
+        $scope.isDisabled = false
+        $window.location.reload()
```

Comment:
Hard reload to get back to the user sign in screen (with appropriate flash message)